File: [5 kyu] Mean Square Error/main.py
def solution(array_a, array_b):
    if not array_a:
        return 0.0

    sum = 0.0

    for i in range(len(array_a)):
        sum += (array_a[i] - array_b[i]) ** 2

    return sum / len(array_a)


def solution(array_a, array_b):
    if not array_a:
        return 0.0

    return sum((x - y) ** 2 for x, y in zip(array_a, array_b)) / len(array_a)

File: [5 kyu] Mean Square Error/main.rs
pub fn solution(array_a: &[i64], array_b: &[i64]) -> f64 {
    if array_a.is_empty() {
        return 0.0;
    }

    array_a
        .iter()
        .zip(array_b.iter())
        .map(|(a, b)| (a - b).pow(2) as f64)
        .sum::<f64>()
        / array_a.len() as f64
}
